### PR TITLE
Add go-root to set Go 1.11

### PR DESCRIPTION
**Pull request by ernestojeda**
Add custom go-root based off new packer builds after merge of https://github.com/edgexfoundry/ci-management/pull/270

Signed-off-by: Ernesto Ojeda <ernesto.ojeda@intel.com>

### CODE REVIEW

**Comment by ernestojeda** [comment]
@MightyNerdEric The local-docker.yml was merged yesterday and it now should support Go 1.11.2. Is it possible to test this change on the Jenkins sandbox?

**Comment by eb-oss** [comment]
> @MightyNerdEric The local-docker.yml was merged yesterday and it now should support Go 1.11.2. Is it possible to test this change on the Jenkins sandbox?

Yes. Please message me on Slack if you need assistance getting the sandbox testing set up.

**Comment by ernestojeda** [comment]
@MightyNerdEric I was able to get the Sandbox job push and it finally built properly. Here is the sandbox passing build: https://jenkins.edgexfoundry.org/sandbox/job/device-sdk-go-master-verify-go/10/console. I had to remove some of the LF pieces causing it to fail like missing Maven settings. But the core functionality of installing Go 1.11.2 and running the make build && make docker has been tested. I am still waiting for the ARM build to complete.

**Comment by ernestojeda** [comment]
ARM success: https://jenkins.edgexfoundry.org/sandbox/job/device-sdk-go-master-verify-go-arm/2/console

File: shell/install_device_sdk_go_deps.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+echo "--> install_device_sdk_go_deps.sh"
+env
```
Comment (ernestojeda) [inline shell/install_device_sdk_go_deps.sh]: Line no. 3 I am going to remove this. This was just for debug.

**Review by eb-oss — APPROVED** [review]

**Pull request merged**